src/libraries/Microsoft.Win32.SystemEvents/ref/Microsoft.Win32.SystemEvents.cs

```diff
@@ -58,23 +58,23 @@ public enum SessionSwitchReason
-        public static event System.EventHandler EventsThreadShutdown { add { } remove { } }
-        public static event System.EventHandler InstalledFontsChanged { add { } remove { } }
+        public static event System.EventHandler? DisplaySettingsChanged { add { } remove { } }
+        public static event System.EventHandler? DisplaySettingsChanging { add { } remove { } }
+        public static event System.EventHandler? EventsThreadShutdown { add { } remove { } }
+        public static event System.EventHandler? InstalledFontsChanged { add { } remove { } }
         [System.ComponentModel.BrowsableAttribute(false)]
         [System.ComponentModel.EditorBrowsableAttribute(System.ComponentModel.EditorBrowsableState.Never)]
         [System.ObsoleteAttribute("This event has been deprecated. https://go.microsoft.com/fwlink/?linkid=14202")]
-        public static event System.EventHandler LowMemory { add { } remove { } }
-        public static event System.EventHandler PaletteChanged { add { } remove { } }
-        public static event Microsoft.Win32.PowerModeChangedEventHandler PowerModeChanged { add { } remove { } }
-        public static event Microsoft.Win32.SessionEndedEventHandler SessionEnded { add { } remove { } }
-        public static event Microsoft.Win32.SessionEndingEventHandler SessionEnding { add { } remove { } }
-        public static event Microsoft.Win32.SessionSwitchEventHandler SessionSwitch { add { } remove { } }
-        public static event System.EventHandler TimeChanged { add { } remove { } }
-        public static event Microsoft.Win32.TimerElapsedEventHandler TimerElapsed { add { } remove { } }
-        public static event Microsoft.Win32.UserPreferenceChangedEventHandler UserPreferenceChanged { add { } remove { } }
-        public static event Microsoft.Win32.UserPreferenceChangingEventHandler UserPreferenceChanging { add { } remove { } }
+        public static event System.EventHandler? LowMemory { add { } remove { } }
+        public static event System.EventHandler? PaletteChanged { add { } remove { } }
+        public static event Microsoft.Win32.PowerModeChangedEventHandler? PowerModeChanged { add { } remove { } }
+        public static event Microsoft.Win32.SessionEndedEventHandler? SessionEnded { add { } remove { } }
+        public static event Microsoft.Win32.SessionEndingEventHandler? SessionEnding { add { } remove { } }
+        public static event Microsoft.Win32.SessionSwitchEventHandler? SessionSwitch { add { } remove { } }
+        public static event System.EventHandler? TimeChanged { add { } remove { } }
+        public static event Microsoft.Win32.TimerElapsedEventHandler? TimerElapsed { add { } remove { } }
+        public static event Microsoft.Win32.UserPreferenceChangedEventHandler? UserPreferenceChanged { add { } remove { } }
+        public static event Microsoft.Win32.UserPreferenceChangingEventHandler? UserPreferenceChanging { add { } remove { } }
```

Comment:
Github won't let me comment in it, but can the sender or the eventhandlerargs in the event handlers be null?

Reply:
Nope.